docs/MAINTAINERS.adoc

```diff
@@ -149,6 +156,140 @@ code reviewers are expected to suggest proper channels for contributors to discu
 * If no consensus can be reached on the mailing list,
   voting at the link:https://www.jenkins.io/project/governance-meeting/[Jenkins Governance Meeting] can be used to get a final decision.
 
+=== Bringing pull requests to closure
+
+==== Motivation
```

Comment:
Maybe it makes sense to also mention that we try to balance value to users and maintenance cost, and do not set a goal to merge all pull requests, especially if they should be rather implemented in plugins.

Reply:
Done in commit a8994a2d12